### PR TITLE
Create frontmatter script

### PR DESCRIPTION
**Pull request by LizBaker**

### CODE REVIEW

File: scripts/utils/getFilesRecursively.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+const getDirectories = (filepath) =>
+  fs.readdirSync(filepath).map(pathJoin(filepath)).filter(isDirectory);
+
+const getFiles = (filepath) =>
+  fs.readdirSync(filepath).map(pathJoin(filepath)).filter(isFile);
+
+const getFilesRecursively = (filepath) =>
+  getDirectories(filepath)
+    .flatMap(getFilesRecursively)
+    .reduce((acc, file) => [...acc, file], getFiles(filepath));
```
Comment (sunnyzanchi) [inline scripts/utils/getFilesRecursively.js:17]: this `reduce` is `concat` 🙂

```suggestion
  getFiles(filepath).concat(
    getDirectories(filepath).flatMap(getFilesRecursively)
  )
```

**Review by sunnyzanchi — APPROVED** [review]
didn't run the script but the code looks good. left one minor comment

**Comment by clarkmcadoo** [comment]
fantastic

**Review by clarkmcadoo — APPROVED** [review]

File: scripts/utils/getFilesRecursively.js (hunk 1 above)
Comment (LizBaker) [reply scripts/utils/getFilesRecursively.js:17]: ooh good find

**Pull request merged**